clients/src/main/java/org/apache/kafka/common/record/TimestampType.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.common.record;
+
+/**
+ * The timestamp type of the records.
+ */
+public enum TimestampType {
+    NoTimestampType(-1, "NoTimestampType"), CreateTime(0, "CreateTime"), LogAppendTime(1, "LogAppendTime");
```

Comment:
Shouldn't these be called `NO_TIMESTAMP_TYPE`, `CREATE_TIME` and `LOG_APPEND_TIME` since this is Java code? I personally prefer the Scala convention (which is what we are using here), but it is inconsistent with all the Java enums I looked at.